src/docs/admin/references/ServiceNowReference.adoc

```diff
@@ -0,0 +1,560 @@
+*Product Name:* ServiceNow +
+*Product Version:* Quebec +
+*Compatibility:* Tokyo
+
+==== Overview
+
+The ServiceNow integration provides means to establish a bidirectional communication
+between a ProActive server, and a ServiceNow instance.
+
+==== Objectives
+
+The goal is to provide templates and coding elements to help ServiceNow and ProActive engineers to easily integrate both solutions in their pipelines.
+
+It boils down to: +
+
+* Provide reusable code in ServiceNow to query ProActive's REST APIs.
+* Provide workflows in ProActive's catalog to query ServiceNow's REST APIs.
+
+==== Installation
+
+This section describes the available processes to set up the required files in ServiceNow in order to communicate
```

Comment:
The process choice is driven by the architecture we will have in ServiceNow.

Reply:
I think it's better to start with "This section..." since he is introducing a new one. You can replace in order to by to directly: This section describes the available processes to set up the required files in ServiceNow to communicate ...